scss/_mixins.scss

```diff
@@ -17,6 +17,9 @@
 	$border-width: map-get($button-dimensions, border-width);
 	$vertical-padding: 1/2 * (map-get($button-dimensions, height) - map-get($button-dimensions, font-size)) - $border-width;
 
+	background-size: map-get($button-dimensions, background-size);
+	height: map-get($button-dimensions, height);
```

Comment:
This might be better being a `min-height` as the multiple lines buttons on the test page overflow the button with a fixed height.